src/groundStation/system.py

```diff
@@ -645,10 +643,194 @@ def __init__(self):
+                                'Secure Key': '>u4',
+                            }
+                        }
+                    },
                 }
             },
+            
+            'CONFIGURATION': {
+                'port': 9,  # As per EPS docs
+                'subservice': {
+                    'GET_ACTIVE_CONFIG': {
+                        'what': 'Gets config values in active mode for a specific type',
+                        'subPort': 0,
+                        'inoutInfo': {
+                            'args': ['<u2', '<u1'], #id, type_id
+                            'returns': {
+                                'err': '>B',
+                                'type': '<u1',
+                                'Value': 'var' #In command parser it gets the data type from the return value for 'type'
+                            }
+                        }
+                    },   
+                    'GET_MAIN_CONFIG': {
+                        'what': 'Gets config values in main mode for a specific type',
+                        'subPort': 1,
+                        'inoutInfo': {
+                            'args': ['<u2', '<u1'], #id, type_id
+                            'returns': {
+                                'err': '>B',
+                                'type': '<u1',
+                                'Value': 'var' #See comment for active
+                            }
+                        }
+                    },  
+                    'GET_FALLBACK_CONFIG': {
+                        'what': 'Gets config values in fallback mode for a specific type',
+                        'subPort': 2,
+                        'inoutInfo': {
+                            'args': ['<u2', '<u1'], #id, type_id
+                            'returns': {
+                                'err': '>B',
+                                'type': '<u1',
+                                'Value': 'var' #See comment for active
+                            }
+                        }
+                    },
+                    'GET_DEFAULT_CONFIG': {
+                        'what': 'Gets config values in default mode for a specific type',
+                        'subPort': 3,
+                        'inoutInfo': {
+                            'args': ['<u2', '<u1'], #id, type_id
+                            'returns': {
+                                'err': '>B',
+                                'type': '<u1',
+                                'Value': 'var' #See comment for active
+                            }
+                        }
+                    },                                                                                                                      
+                    'SET_CONFIG': {
+                        'what': 'Sets the configuration',
+                        'subPort': 4,
+                        'inoutInfo': {
+                            'args': ['<u2', '<u1', 'var'], #id, type, config
```

Comment:
Why don't we know the size ahead of time?

Reply:
Because it uses the same subport for different sizes. You could send `eps.configuration.set_config(0 0 1)` or `eps.configuration.set_config(126 4 32569)` where the first one is uint8 and the second one is uint32.
I considered having separate commands for each type, but since the support is the same, it would be much more complicated.

Reply:
Okay, I think this is a nice solution then